index.html

```diff
@@ -2825,125 +2827,265 @@ <h3>Working with personal names</h3>
 </aside>
 
 
+<section id="loc_names_field_length">
+<h4>Field length &amp; composition</h4>
+
 	<div class="req" id="loc_names_given_family">
-	<p class="advisement">Check whether you really need to store or access given name and family name separately.</p>
-	<details class="links"><summary>explanations &amp; examples</summary>
+	<p class="advisement">Check whether you really need to store or access given name(s) and family name(s) separately.</p>
+	<details class="links"><summary>source</summary>
 	<p><a href="https://www.w3.org/International/questions/qa-personal-names#singlefield">To split or not to split?</a> in <cite>Personal Names Around the World</cite>.</p>
 	</details>
+	<details class="links"><summary>examples</summary>
+	<p><a href="https://www.w3.org/International/questions/qa-personal-names#examples">Examples of differences</a> in <cite>Personal Names Around the World</cite>.</p>
+	</details>
 	</div>
 
+<p>Names around the world differ greatly in composition and the order of components (see <a href="https://www.w3.org/International/questions/qa-personal-names">Personal names around the world</a>). This can create difficulties when you try to split a person's name into smaller parts for storage in a database and then retrieve them. Difficulties include understanding which part of a person's name belongs in which database field (especially when there are more or fewer parts than fields in the database), and dealing with ordering of name parts when retrieving someone's name from the database for actual use.</p>
```

Comment:
It may not involve reconstruction. A developer may try to just retrieve the family name, say for sorting.

Reply:
Fair enough. I would probably still change this:

> This can create difficulties when you try to split a person's name into smaller parts for storage in a database and then retrieve them.

to something like:

> This can create difficulties when you try to split a person's name into smaller parts for storage and then retrieve them for use later.

(I removed "in a database" as too specific and added "for use later" just because I think it reads more clearly)